Task10/task10.css

```diff
@@ -0,0 +1,244 @@
+    margin: auto 40px;
+    background-repeat: no-repeat;
+}
+.text_button{
+    text-align: center;
+    padding-top: 10px;
+    font-size: 22px;
+    color: rgba(68, 68, 68, 0.82);
+}
+.main{
+    margin-top: 170px;
+}
+.navigation_Home{
+    background-position: 0 0;
+}
+.navigation_About{
+    background-position: -55px 0;
+}
+.navigation_Work{
+    background-position: -110px 0;
+}
+.navigation_Blog{
+    background-position: -165px 0;
+}
+.navigation_Contact{
+    background-position: -220px 0;
+}
+.image-orange{
+    background-image: url("img/big-orange-bg.png");
+    height: 500px;
+    background-repeat: no-repeat;
+    background-position: center center;
+    background-size: cover;
+    display: flex;
+    align-items: center;
+    border-bottom: 4px solid rgb(252, 163, 0);
+    border-top: 4px solid rgb(252, 163, 0);
+}
+.question-block{
+    margin-left:200px;
+}
+.question-main{
+    margin: 20px;
+    font-size: 50px;
+    padding: 10px 20px;
+}
+.question-rabbits{
+    background-color: rgba(29, 29, 29, 0.55);
+    color: rgb(252, 163, 0);
+    font-family: BebasNeueRegular;
+    font-weight: 800;
+    letter-spacing: 2px;
+}
+.question-oranges{
+    background-color: rgba(255, 157, 0, 0.55);
+    color: rgba(7, 5, 0, 0.48);
+    font-family: BebasNeueRegular;
+    font-weight: 800;
+    letter-spacing: 2px;
+}
+.column-content{
+    display: flex;
+    justify-content: center;
+    flex-wrap: wrap;
+}
+.column{
+    width: 350px;
+    text-align: justify;
+    font-family: Calibri;
+    font-weight: 600;
+    font-size: 22px;
+    padding: 20px;
+}
+.read_more{
+    text-align: right;
+    margin-top:10px;
+    font-style: italic;
+    color: #fb9d01;
+}
+h3{
+    color: #fb9d01;
+    text-align: center;
+    font-size: 50px;
+    font-weight: 700;
+    font-family: StMarie-Thin;
+    display: flex;
+    justify-content: center;
+}
+.span-h3{
+    border-bottom: 2px solid #fb9d01;
+}
+.about_position{
+    display: flex;
+    justify-content: center;
+    flex-wrap: wrap;
+}
+.column-position{
+
+}
+.person_head{
+    text-align: center;
+    font-family: Calibri;
+    font-size: 22px;
+    font-weight: 800;
+}
+.position-color{
+    color: #fb9d01;
+}
+.person_image{
+    width: 350px;
+    height: 350px;
+    border-radius: 50%;
+    border: 8px solid #fb9d01;
+    margin: 10px;
+}
+.person_description{
+    text-align: justify;
+    max-width: 350px;
+    font-family: Calibri;
+    font-size: 20px;
+    font-weight: 800;
+    margin: 20px auto;
+}
+.entry_field{
+
+}
+.element-form{
+    display: flex;
+    justify-content: flex-end;
+    padding: 5px 0;
+}
+.lable{
+    font-size: 30px;
+    color: #fb9d01;
+    width: 140px;
+}
+.input{
+    height: 30px;
+    width: 90%;
+    -webkit-box-shadow: inset 5px 5px 0px -2px rgba(0,0,0,0.21);
+    box-shadow: inset 5px 5px 0px -2px rgba(0,0,0,0.21);
+    border: 0;
+    background-color: #f6f6f6;
+}
+.textarea{
+    min-height: 62px;
+}
+.submit{
+    width: 200px;
```

Comment:
Will be better to use padding instead of width and height. 
Button is inline element so use display: inline-block;

Reply:
done